languages/common-lisp/exercises/concept/socks-and-sexprs/.docs/instructions.md

```diff
@@ -1,17 +1,17 @@
-You! Yes you, brave lisper! [Lenny the lisp alien][alien] needs your help! He
-wants to go hang out with all the other lisp aliens, but his parents have told him
-that he's not going anywhere until his room is clean.
+You! Yes you, brave lisper! [Lenny the lisp alien][alien] needs your help! Lenny
+wants to go hang out with all the other lisp aliens, but their parents have told him
```

Comment:
Looks like we have a sneaky "him" that got left behind here!

Reply:
And a "he's" on the next line :)

Reply:
Ye Gods! This is getting embarrassing!